src/plugins/timeConductor/TimePopupFixed.vue

```diff
@@ -215,96 +232,98 @@ export default {
         return false;
       }
     },
+    clearAllValidation() {
+      Object.keys(this.inputValidityMap).forEach(this.clearValidation);
+    },
+    clearValidation(refName) {
+      const input = this.getInput(refName);
+
+      input.setCustomValidity('');
+      input.title = '';
+    },
+    onInput(refName) {
```

Comment:
Please give this a name that describes what it does, not the event that it handles. eg. `clearValidationAndRevalidate`